DependencyInjection/Configuration.php

```diff
@@ -413,6 +413,7 @@ public function getConfigTreeBuilder()
                                     ->scalarNode('hostname')->end()
                                     ->scalarNode('port')->defaultValue(12201)->end()
                                     ->scalarNode('chunk_size')->defaultValue(1420)->end()
+                                    ->booleanNode('ignore_error')->defaultValue(false)->end()
```

Comment:
logging errors should never break the app , esp. in production. @Seldaek does monolog (bundle) not catch processing errors automatically? If not, shouldn't handler error be catched always? This is similar to caching where the PSR says, caching errors should never throw exceptions.

Reply:
And if this get's only handled on a handler-by-handler basis, this ignore_error option should IMO default to `%kernel.debug%`.

Reply:
The best "monolog" way to handle this is to wrap handlers in a WhatFailureGroupHandler which suppresses any exception.. That can apply to any handler instead of being gelf-specific